src/main/java/org/apache/commons/collections4/bag/UnmodifiableBag.java

```diff
@@ -128,6 +129,11 @@ public boolean remove(final Object object) {
         throw new UnsupportedOperationException();
     }
 
+    @Override
```

Comment:
I'm also thinking that all new public methods should have a `@since 4.4` Javadoc tag.